implementations/python/sugar-lift-py-tests/tests/fixtures/with_v2_laws/enrollment_cross_file.py

```
@@ -0,0 +1,6 @@
+from enrollment_helper import resolve
+class Sugar: pass
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)